components/molecule/avatar/src/index.scss

```diff
@@ -0,0 +1,54 @@
+@import '~@s-ui/theme/lib/index';
+@import '~@s-ui/react-atom-image/lib/index';
+@import '~@s-ui/react-atom-skeleton/lib/index';
+@import './AvatarFallback/index';
+@import './AvatarBadge/index';
+
+$s-avatar: (
+  xlarge: 96px,
+  large: 64px,
+  medium: 48px,
+  small: 32px
+) !default;
+
+$fs-avatar-name: (
```

Comment:
Is this `font-size`? if so, the name should be `fz` instead of `fs`
Side note: In general, I would suggest to apply font-sizes as variables i.e. `large: $fz-l`, but in this case it's ok to put hardcoded integers.

Reply:
Change made!